data_steward/validation/main.py

```diff
@@ -1010,7 +1010,18 @@ def process_hpo_copy(hpo_id):
             name: str = item['name']
             full_name: str = f'{hpo_id}/{hpo_bucket.name}/{name}'
             hpo_blob = hpo_bucket.get_blob(name)
-            hpo_bucket.copy_blob(hpo_blob, drc_bucket, full_name)
+            dest_blob = drc_bucket.blob(full_name)
+
+            rewrite_token = False
+            while True:
+                rewrite_token, bytes_rewritten, bytes_to_rewrite = dest_blob.rewrite(
+                    hpo_blob, token=rewrite_token)
+                logging.info(
+                    f"{full_name}: Copied: {bytes_rewritten}/{bytes_to_rewrite} bytes."
+                )
+
+                if not rewrite_token:
+                    break
```

Comment:
Can this be in the gcloud/gcs.py client as a custom function? We have it just for this purpose. Also, maybe we can use gsutil rsync as long term since it only copies if the checksum is different.

Reply:
Thanks for the comment! It all makes sense. Regarding `rsync`, I created an enhancement ticket, DC-3771, where we can implement it as a long-term solution.